template/index.html

```diff
@@ -328,25 +327,25 @@ <h3>Events</h3>
                 </div>
                 <div class="LearnGo-eventBody">
                     <div class="LearnGo-eventDate">
-                        <p>Nov 15, 2019</p>
-                        <p>Leipzig,  Germany</p>
+                        <p>2019年11月15日</p>
+                        <p>德国,莱比锡</p>
                     </div>
                     <div class="LearnGo-eventText">
                         <h4 class="LearnGo-eventName">
-                            <a href="https://www.meetup.com/Leipzig-Golang/events/265955691">Leipzig Golang and Cloud Native Meetup #8</a>
+                            <a href="https://www.meetup.com/Leipzig-Golang/events/265955691">莱比锡Golang与云原生技术沙龙#8</a>
                         </h4>
                         <p class="LearnGo-eventDescription">
-                            We enjoy using Go and we are curious about the language, its features and applications, especially in the context of cloud computing and infrastructure.<br/>
-                            Our goal is to establish a place, where people from all backgrounds interested in the Go programming language and various cloud technologies can meet, learn, share experiences, present, discuss and network.<br/>
-                            We aim to be a welcoming community, respecting Gopher values, as outlined here: <a href="https://golang.org/conduct" rel="nofollow">https://golang.org/conduct</a>.<br/>
+                            我们享受使用Go编程，对Go语言本身及它的特性、应用充满好奇，尤其是与云计算、架构相关的内容。 <br/>
+                            我们的目标是建立一个组织，让不同背景但对Go编程语言和各种云技术感兴趣的人，能在这里聚会、学习、分享经验、表达、讨论、建立人际网络。 <br/>
+                            我们的愿景是成为一个受欢迎的、尊重Gopher价值的社区，正如在这里所强调的: <a href="https://golang.org/conduct" rel="nofollow">https://golang.org/conduct</a>.<br/>
```

Comment:
```suggestion
                            我们的愿景是成为一个受欢迎的、尊重Gopher价值的社区，正如在这里所强调的：<a href="https://golang.org/conduct" rel="nofollow">https://golang.org/conduct</a>。<br/>
```

这个翻译完的语法怪怪的，不太像正儿八经我们日常的说话方式

不过没事，到时候我再全部统筹一遍吧